src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -270,10 +270,12 @@ const mapStateToProps = state => {
     isInMVI: isInMVISelector(state),
     isLOA3,
     shouldFetchDirectDepositInformation,
+
     shouldShowDirectDeposit:
```

Comment:
All `shouldShowDirectDeposit` is used for is to determine whether or not to show the item in the navigation.

Reply:
🎉